assignments/lambda-classes.js

```diff
@@ -1 +1,115 @@
 // CODE here for your Lambda Classes
+class Person {
+    constructor(attributes) {
+        this.name = attributes.name;
+        this.age = attributes.age;
+        this.location = attributes.location;
+        this.gender = attributes.gender;
+
+    }
+    speak() {
+        //console.log(`Object was removed from the game.`);
+        return `Hello my name is Fred, I am from Bedrock`;
+    }
+}
+
+
+class Instructor extends Person {
+    constructor(InstrucAttributes) {
+        super(InstrucAttributes);
+        this.specialty = InstrucAttributes.specialty;
+        this.favLanguage = InstrucAttributes.favLanguage;
+        this.catchPhrase = InstrucAttributes.catchPhrase;
+    }
+    demo(subject) {
+        return `Today we are learning about ${subject}`;
+    }
+
+    grade(student, subject) {
+        `${student.name} receives a perfect score on ${subject}`;
+    }
+
+}
+
+class student extends Person {
+    constructor(studentAttributes) {
+        super(studentAttributes);
+        this.previousBackground = studentAttributes.previousBackground;
+        this.className = studentAttributes.className;
+        this.favSubjects = studentAttributes.favSubjects;
+
+    }
+
+    listsSubjects() {
+        return this.favSubjects.forEach(favSubjects => {
+            console.log(favSubjects);
+        });
+    }
+
+    PRAssignment(subject) { `${student.name} has submitted a PR for ${subject}` }
```

Comment:
Since your only have the 1 parameter in this method, `student.name` wouldn't work since it's not being passed. You would use `this.name` to access the students name in this case.